cmd/process-agent/command/command.go

```diff
@@ -159,23 +156,6 @@ func BootstrapConfig(path string, oneshotCommand bool) error {
 	)
 }
 
-// setHostMountEnv sets HOST_PROC and HOST_SYS mounts if applicable in containerized environments
-func setHostMountEnv() {
-	// Set default values for proc/sys paths if unset.
-	// Don't set this is /host is not mounted to use context within container.
-	// Generally only applicable for container-only cases like Fargate.
-	if !config.IsContainerized() || !util.PathExists("/host") {
-		return
-	}
-
-	if v := os.Getenv("HOST_PROC"); v == "" {
-		os.Setenv("HOST_PROC", "/host/proc")
```

Comment:
What about setting this?

Reply:
Ah, sorry should have been clear on this. There's actually a duplicate override of `HOST_PROC` in `InitConfig`, it's right above where I override `HOST_SYS`. Based off @hush-hush's comment, I will actually move this back into the process-agent however, since I didn't realize the core agent uses gopsutil as well.